testsuite/manualmode/src/test/java/org/wildfly/core/test/standalone/mgmt/HTTPSManagementInterfaceTestCase.java

```diff
@@ -265,6 +317,19 @@ private static void serverSetup(ManagementClient managementClient) throws Except
         CoreUtils.applyUpdate(operation, client);
     }
 
+    private static void changeSecurityRealmForHttpMngmntInterface(String securityRealmName) throws Exception {
+        //ModelControllerClient client = controller.getClient().getControllerClient();
+        try (ModelControllerClient client = getNativeModelControllerClient()) {
+            ModelNode operation = createOpNode("core-service=management/management-interface=http-interface",
+                    ModelDescriptionConstants.WRITE_ATTRIBUTE_OPERATION);
+            operation.get(NAME).set("security-realm");
+            operation.get(VALUE).set(securityRealmName);
+            CoreUtils.applyUpdate(operation, client);
+
+            reloadServer();
```

Comment:
There's no need for the write-attribute or reload if a read-attribute shows the attribute already has the desired value.